README.md

```diff
@@ -229,7 +229,7 @@ The Web UI can be found at http://localhost:9000 while the SMTP port will be on
 
 ### Support IPv6
 
-In case you are deploying to a cluster that uses only IPv6. You can use a custom command to pass a parameter to the Next.js start command
+In case you are deploying to a cluster that uses only IPv6,You can use a custom command to pass a parameter to the Next.js start command
```

Comment:
"[...] uses only **IPv6,You** can use a custom [...]" should be "[...] uses only **IPv6, you** can use a custom [...]".

Reply:
There should also be a full stop at the end of the sentence.